app/jobs/submit_scan_request_job.rb

```diff
@@ -0,0 +1,44 @@
+class SubmitScanRequestJob < ActiveJob::Base
+  queue_as :default
+
+  def perform(request)
+    response = client.get('func_request_webservice_new.make_request', request_params(request)) do |req|
```

Comment:
Unused block argument - `req`. You can omit the argument if you don't care about it.